core/src/com/unciv/ui/utils/GeneralPlatformSpecificHelpers.kt

```diff
@@ -11,7 +11,12 @@ interface GeneralPlatformSpecificHelpers {
      *  @param allow `true`: allow all orientations (follows sensor as limited by OS settings)
      *          `false`: allow only landscape orientations (both if supported, otherwise default landscape only)
      */
-    fun allowPortrait(allow: Boolean)
+    fun allowPortrait(allow: Boolean) {}
 
     fun isInternetConnected(): Boolean
+
+    /**
+     * Notifies the user that it's their turn while the game is running
+     */
+    fun notifyTurnStarted() {}
```

Comment:
That's allowed? I though default implementations necessitate abstract class instead of interface... And this should definitely stay a pure interface.

Reply:
Ah now I remember. Of course. It's _fields_ that must stay abstract.